app/assets/javascripts/take-whole-pot-calculator.js

```diff
@@ -3,7 +3,7 @@
 
   var takeWholePotCalculator = {
     init: function() {
-      var $calculator = $('.js-take-whole-pot-calculator form');
+      var $calculator = $('.js-calculator--take-whole-pot form');
```

Comment:
This js actually [never runs](https://github.com/guidance-guarantee-programme/pension_guidance/blob/calculators/css-naming/app/assets/javascripts/application.js) in the current version, so might be simpler just to remove as we're refactoring anyway?